### PR TITLE
@alloy -> add top rule to nave bar

### PR DESCRIPTION
**Pull request by 1aurabrown**
closes https://github.com/artsy/eigen/issues/306

### CODE REVIEW

File: Artsy/Classes/View Controllers/ARTopMenuViewController.m [hunk 1]
```diff
@@ -109,7 +115,7 @@ - (void)viewDidLoad
     [tabContainer.layer addSublayer:divider];
     _divider = divider;
 
-    self.tabHeightConstraint = [[tabContainer alignBottomEdgeWithView:self.view predicate:@"0"] firstObject];
+    self.tabHeightConstraint = [[tabContainer alignBottomEdgeWithView:self.view predicate:@"0"] lastObject];
```
Comment (1aurabrown) [inline Artsy/Classes/View Controllers/ARTopMenuViewController.m]: I think the constraint that gets added is appended to the end of the array of constraints

Comment (1aurabrown) [inline Artsy/Classes/View Controllers/ARTopMenuViewController.m]: But I vaguely remember that we determined at some point that this doesn't matter

Comment (1aurabrown) [inline Artsy/Classes/View Controllers/ARTopMenuViewController.m]: It doesn't matter in this case, as the constraint array returned will only have one object, but we use lastObject everywhere else so I'll keep this in for consistency.

Comment (alloy) [inline Artsy/Classes/View Controllers/ARTopMenuViewController.m]: Fair.

**Comment by alloy** [comment]
+1 for sneaking in the CHANGELOG of the previous change :D

**Pull request merged**